prime-router/settings/organizations.yml

```diff
@@ -2067,6 +2067,46 @@
   jurisdiction: STATE
   stateCode: PA
   receivers:
+    - name: elr
+      organizationName: pa-phd
+      topic: covid-19
+      customerStatus: active
+      jurisdictionalFilter: [ "matches(ordering_facility_state, PA)" ]
```

Comment:
Do we also want to add the patient state here, or did PA explicitly not ask for that?